modules/error_log/cog.py

```diff
@@ -0,0 +1,69 @@
+import discord
+from discord.ext import commands
+from discord import logging
+import traceback
+from modules.error_log.utils import get_discord_obj, get_logs, log_to_file
+
+
+class ErrorLogCog(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(name="logs")
+    async def logs(self, ctx, *args):
+        """Show recent logs from err.log"""
+        # log in console that a ping was received
+        print('Executing command "logs".')
+        # reply with last 1200 characters of log file
+        await ctx.send(f"```...{get_logs('err.log')[-1200:]}```")
+
+    @commands.Cog.listener()
+    async def on_message(self, message):
```

Comment:
Is there a reason why this is not a command? I think it makes more sense as a command

Reply:
As a command, it would trigger the `on_command_error` but the purpose was to test `on_error` without triggering `on_command_error`.

It can be removed if necessary since it is only for testing.